templates/rancher-airgap/overlay/opt/corral/rancher/preflight.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+
+apt-get update || true
+
+apt install -y jq || true
```

Comment:
Still an unsolved problem, just remarking that relying on apt is going to make our infra very brittle. It may even be a good idea to install jq from source in this case.

Reply:
Yeah that's probably the best option.